packages/chrome.rb

```diff
@@ -4,15 +4,20 @@ class Chrome < Package
   description 'Google Chrome is a fast, easy to use, and secure web browser.'
   homepage 'https://www.google.com/chrome'
   compatibility 'x86_64'
-  @_ver = '87.0.4280.141'
+  @_ver = '88.0.4324.182'
   @_deb = "google-chrome-stable_#{@_ver}-1_amd64.deb"
   version @_ver
+  
+  binary_url ({
+    x86_64: "https://dl.google.com/linux/deb/pool/main/g/google-chrome-stable/#{@_deb}"
+  })
+  binary_sha256 ({
+    x86_64: 'ab00e9412f5f20e30c7db5dc987473248f4adf9ebf2c3f928ef62e1ffb104fe6'
+  })
```

Comment:
Technically, this would be considered a source since we have to move files around to get it working on ChromeOS.  A true binary would include Chromebrew metadata and simply be installed by extracting all the files.